clouds/aws/aws_roles/aws_roles.py

```diff
@@ -22,30 +20,21 @@ def iam_client(region=DEFAULT_AWS_REGION):
 
 
 def create_or_update_role_policy(
-    role_name, policy_name, policy_document_path, region=DEFAULT_AWS_REGION
+    role_name, policy_name, policy_document, region=DEFAULT_AWS_REGION
 ):
     """
     Create a new policy role or update an existing one.
 
     Args:
         role_name (str): role policy name
         policy_name (str): policy name
-        policy_document_path (str): path to Json file that holds the policy documents
+        policy_document (str): loaded json file that holds the policy documents
```

Comment:
`loaded` what do you mean by this?
Maybe  `policy documents as Json content`  ?

Reply:
Done